### PR TITLE
Fix unicode issue, minor improvement to use app_config.app_titile

### PR DESCRIPTION
**Pull request by cyberstar**
if translation was provided for app_confug.app_title use it, otherwise fall back to namespace.

### CODE REVIEW

File: aldryn_events/cms_appconfig.py [hunk 1]
```diff
@@ -52,3 +52,11 @@ class EventsConfig(TranslatableModel, AppHookConfig):
     placeholder_events_registration_footer = PlaceholderField(
         'events_registration_footer',
         related_name='aldryn_events_registration_footer')
+
+    def __unicode__(self):
```
Comment (mkoistinen) [inline aldryn_events/cms_appconfig.py]: Shouldn't this be def __str__(self) with the python_2_unicode_compatible decorator on the class?

File: aldryn_events/models.py [hunk 2]
```diff
@@ -177,12 +179,10 @@ def get_title(self):
         return self.safe_translation_getter('title', any_language=True)
 
     def __unicode__(self):
```
Comment (mkoistinen) [inline aldryn_events/models.py]: Same here.

**Comment by mkoistinen** [comment]
LGTM, but can we address my comments before merging?

**Comment by fmarco** [comment]
LGTM

**Comment by cyberstar** [comment]
rebased

**Pull request merged**